packages/contracts-bedrock/contracts/test/AddressAliasHelper.t.sol

```diff
@@ -5,7 +5,7 @@ import { Test } from "forge-std/Test.sol";
 import { AddressAliasHelper } from "../vendor/AddressAliasHelper.sol";
 
 contract AddressAliasHelper_Test is Test {
-    function test_fuzz_roundtrip(address _address) external {
+    function testFuzz_applyAndUndo_succeeds(address _address) external {
```

Comment:
TBH I'm now leaning towards removing the `_succeeds` postfix, but I'd like to get these changes in as is, then we could easily cut it out in a subsequent PR.